cmd/utils/flags.go

```diff
@@ -231,6 +231,10 @@ var (
 		Name:  "db.leveldb.no-buffer-pool",
 		Usage: "Disables using buffer pool for LevelDB's block allocation",
 	}
+	LevelDBNoPerformanceCheckFlag = cli.BoolFlag{
+		Name:  "db.leveldb.no-perf-check",
+		Usage: "Disables performance checking of LevelDB's read and write operations",
+	}
```

Comment:
How about adding a word "Metric" in the expanation of the flag?
```suggestion
	LevelDBNoPerformanceCheckMetricFlag = cli.BoolFlag{
		Name:  "db.leveldb.metrics.no-perf-check",
		Usage: "Disables performance checking metric of LevelDB's read and write operations",
	}
```

Reply:
@aidan-kwon I think "Performance Check" already has meaning of "metrics", how do you think?

Reply:
"Performance Check" reminds me of profiling something. Anyway... it's not a big deal :)